Cargo.toml

```diff
@@ -132,22 +132,22 @@ tempfile = { workspace = true }
 
 [features]
 plugin = [
-  "nu-plugin",
-  "nu-cli/plugin",
-  "nu-parser/plugin",
-  "nu-command/plugin",
-  "nu-protocol/plugin",
-  "nu-engine/plugin",
+	"nu-plugin",
+	"nu-cli/plugin",
+	"nu-parser/plugin",
+	"nu-command/plugin",
+	"nu-protocol/plugin",
+	"nu-engine/plugin",
 ]
 default = ["default-no-clipboard", "system-clipboard"]
 # Enables convenient omitting of the system-clipboard feature, as it leads to problems in ci on linux
 # See https://github.com/nushell/nushell/pull/11535
 default-no-clipboard = [
-  "plugin",
-  "which-support",
-  "trash-support",
-  "sqlite",
-  "mimalloc",
+	"plugin",
+	"which-support",
+	"trash-support",
+	"sqlite",
+	"mimalloc",
```

Comment:
This should probably be reverted to the formatting we use in the rest of the tomls, also to keep the `git blame` easier to read.